hoot-services/src/main/java/hoot/services/utils/DbUtils.java

```diff
@@ -841,7 +841,7 @@ public static String getJobIdByTask(String taskInfo) {
         return jobId;
     }
 
-    public static String getJobBbox(String jobId) {
+    public static String getJobBounds(String jobId) {
         String foundId = createQuery()
             .select(Expressions.stringTemplate("tags->'bbox'"))
```

Comment:
Does this bbox key need to change?  or do we need to add a bounds key and select for either to be backwards compatible.